src/components/cc-domain-management/cc-domain-management.js

```diff
@@ -0,0 +1,819 @@
+              </div>
+            </cc-badge>
+          ` : ''}
+
+          ${isHttpOnly ? html`
+            <cc-badge intent="warning" weight="outlined">
+              <div class="badge-content">
+                <cc-icon .icon=${iconHttpOnly} a11y-name=${i18n('cc-domain-management.list.badge.http-only.alt')}></cc-icon>
+                ${i18n('cc-domain-management.list.badge.http-only.text')}
+              </div>
+            </cc-badge>
+          ` : ''}
+        </div>
+
+        <div class="actions">
+          ${!isPrimary ? html`
+            <cc-button 
+              class="mark-primary"
+              primary
+              outlined
+              ?disabled=${isMarkingPrimaryDisabled}
+              ?waiting="${domainItemStateType === 'marking-primary'}"
+              a11y-name="${i18n('cc-domain-management.list.btn.primary.a11y-name', { domain: hostWithWildcard })}"
+              .icon=${iconPrimary}
+              hide-text
+              circle
+              @cc-button:click=${this._onMarkPrimary(domainInfo)}
+            >
+              ${i18n('cc-domain-management.list.btn.primary.text')}
+            </cc-button>
+          ` : ''}
+          <cc-button
+            class="delete-domain"
+            danger 
+            outlined
+            ?disabled="${domainItemStateType === 'marking-primary'}"
+            ?waiting="${domainItemStateType === 'deleting'}"
+            a11y-name="${i18n('cc-domain-management.list.btn.delete.a11y-name', { domain: hostWithWildcard })}"
+            .icon=${iconDelete}
+            hide-text
+            circle
+            @cc-button:click=${this._onDelete(domainInfo)}
+          >
+            ${i18n('cc-domain-management.list.btn.delete.text')}
+          </cc-button>
+        </div>
+      </div>
+    `;
+  }
+
+  /**
+   * @param {string} cnameRecord
+   * @param {string[]} aRecords
+   * @returns {TemplateResult}
+   */
+  _renderDnsInfo (cnameRecord, aRecords) {
+    return html`
+      <cc-block-section>
+        <div slot="title">${i18n('cc-domain-management.dns.cname.heading')}</div>
+        <div slot="info">${i18n('cc-domain-management.dns.cname.desc')}</div>
+        <cc-input-text hidden-label label="${i18n('cc-domain-management.dns.cname.label')}" readonly clipboard value=${cnameRecord}></cc-input-text>
+      </cc-block-section>
+      <cc-block-section>
+        <div slot="title">${i18n('cc-domain-management.dns.a.heading')}</div>
+        <div slot="info">${i18n('cc-domain-management.dns.a.desc')}</div>
+        <div class="a-records">
+          ${aRecords.map((aRecord, index) => html`
+            <cc-input-text hidden-label label="${i18n('cc-domain-management.dns.a.label', { index: index + 1 })}" readonly clipboard value=${aRecord}></cc-input-text>
+          `)}
+        </div>
+      </cc-block-section>
+    `;
+  }
+
+  static get styles () {
+    return [
+      accessibilityStyles,
+      linkStyles,
+      css`
```

Comment:
same as formatting, when rebasing the CSS properties ordering rules changed and I decided not to fix issues before the review phase is over to avoid cluttering the diffs